data/gtk-style.css

```diff
@@ -804,6 +804,17 @@ ClubhouseViewMainLayer #banister {
   background-image: url("resource:///com/hack_computer/Clubhouse/images/banister.png");
   transition: background-image 200ms linear; }
 
+ClubhouseViewMainLayer .extension-button {
+  color: white;
+  text-shadow: none;
+  border: 0;
+  margin: 0;
+  padding: 3px;
+  border-radius: 3em;
+  box-shadow: 1px 1px 3px black;
+  background-color: red;
```

Comment:
We lack a designer right now, but plain "red" is rarely used in UIs. Anyways, grepping the CSS is not the first "red" we have. For instance, bootstrap uses [#d9534f](https://github.com/twbs/bootstrap-sass/blob/master/assets/stylesheets/bootstrap/_variables.scss#L22) and adwaita uses [#e01b24](https://gitlab.gnome.org/GNOME/gtk/-/blob/master/gtk/theme/Adwaita/_colors.scss#L32). Please use any of those, but not plain red as it hurts users eyes :)

Reply:
I've changed the red with the adwaita color.